docs/changes.rst

```diff
@@ -163,112 +172,111 @@ details of the changes themselves and the reasons for the changes.
   pywbem.config. Specifically, renamed the following modules by prepending
   an underscore character: cim_constants.py, cim_http.py, cim_obj.py,
   cim_operations.py, cim_types.py, cim_xml.py, exceptions.py, mof_compiler.py,
   moflextab.py, mofparsetab.py, tupleparse.py, tupletree.py.
+  Using these sub-namespaces had been deprecated in pywbem 0.8.0.
 
   This change is compatible for users that followed the recommendation
   to import only the symbols from the pywbem namespace. Users that imported
   symbols from these sub-namespace should now import them from the pywbem
   namespace. If you miss a symbol in the pywbem namespace, it was likely a
   symbol that is not part of the public pywbem API. (See issue #1925)
 
-* Removed the `add_method_callback()` method and the `methods` property
-  from the `FakeWBEMConnection` class.This has been replaced by
-  the user-defined provider concept where the user defines and registers a
-  subclass to the class MethodProvider which implements the InvokeMethod
-  responder in that user-defined provider. (See issue #2062).
+* Mock WBEM Server (experimental):
 
-* Added pywbem_mock.config.py variables to control use of the IncludeQualifiers
-  and IncludeClassOrigin arguments on the GetInstance and EnumerateInstances
-  operations.  The default is not to ignore the request variables and
-  never return either attribute in the return instances.  This may be
-  modified by modifying the config variables. ()See issue # 2065)
-
-* Removed the `FakedWBEMConnection` init argument `conn_lite`. The lite mode
-  turned out too simplistic for mock testing and of no real value, while
-  adding complexity.  (See issue #1959)
+  * Removed the `add_method_callback()` method and the `methods` property
+    from the `FakedWBEMConnection` class. This has been replaced by
+    the user-defined provider concept where the user defines and registers a
+    subclass to the class MethodProvider which implements the InvokeMethod
+    responder in that user-defined provider. (See issue #2062).
+
+  * Removed the `conn_lite` init parameter of `FakedWBEMConnection`. The lite
+    mode turned out too simplistic for mock testing and of no real value, while
+    adding complexity.  (See issue #1959)
+
+  * Changed the logging behavior of the MOF compilation methods
+    `FakedWBEMConnection.compile_mof_string()`, `compile_mof_file()` and
+    `compile_dmtf_schema()` to be able to do no logging, by specifying `None` for
+    the `log_func` init parameter of `MOFCompiler`. This is now the default.
+
+    MOF compile errors no are longer printed to stdout by default. To continue
+    printing the MOF compile errors to stdout, print the exception in your code.
+    (See issue #1997)
+
+  * Added pywbem_mock.config.py variables to control use of the IncludeQualifiers
+    and IncludeClassOrigin arguments on the GetInstance and EnumerateInstances
+    operations.  The default is not to ignore the request variables and
```

Comment:
Propose to replace the text starting with "The default is not to" with the following:

The default is to ignore the request variables and never return either attribute in the return instances whereas, in previous vesions the request variables determined whether they were returned. This may be changed by modifying the config variables however, the behavior of these request parameters was inconsistent between versions of the DMTF specification so the user should not depend on them. (See issue # 2065)

Reply:
DONE - see new text in my comment further up.